test-vectors/src/peg_in.rs

```diff
@@ -0,0 +1,54 @@
+use bitcoin::{
+    opcodes, script::Builder, OutPoint, Script, Sequence, Transaction, TxIn, TxOut, Witness,
+};
+
+use crate::utils::generate_test_vector;
+
+pub fn generate_peg_in_test_vector() -> Transaction {
+    let input = TxIn {
+        previous_output: OutPoint::null(),
+        script_sig: Script::empty().into(),
+        sequence: Sequence(65535),
+        witness: Witness::new(),
+    };
+
+    let input = vec![input];
+
+    // Random generated wallet address
+    let stacks_address = [1u8; 20];
+    let current_peg_wallet = [4; 32];
+
+    let output = generate_peg_in_request_output(stacks_address, current_peg_wallet);
+
+    generate_test_vector(input, output)
+}
+
+fn generate_peg_in_request_output(
```

Comment:
The peg in operation is not a request in the current design.
```suggestion
fn generate_peg_in_output(
```